src/index.js

```diff
@@ -18,17 +71,43 @@ function loginHandler() {
 function login(customerCheck) {
   if($("#username-input").val() === 'manager'
   && $("#password-input").val() === 'overlook2019') {
-    window.location = "./manager-page.html";
+    managerPageHandler(date)
   } else if (customerCheck.includes($("#username-input").val())
   && $("#password-input").val() === 'overlook2019') {
-    window.location = "./customer-page.html";
+    customerPageHandler();
   } else {
     errorMessageHandling();
   }
 }
 
-function logout() {
-  window.location = "./index.html";
+function customerPageHandler() {
+  id = Number($("#username-input").val().split('r')[1]);
+  addCustomerHTML();
+  instantiateCustomer(id);
+  addUserRoomBookings(id);
+  $("#dollars-spent").text(`$${customer.getUserTotalSpent(id)}`)
+}
+
```

Comment:
As you progress through you may want another page handler for when the user books a new room to update their bookings history section and money spent. Or you could have a function that just appends those room details to the already existing list. Just a thought!